src/main/java/com/partycipate/Partycipate/dto/RegionUser.java

```diff
@@ -0,0 +1,7 @@
+package com.partycipate.Partycipate.dto;
+
+public interface RegionUser {
```

Comment:
make this a dto (public class) please with getter, setter & constructor so that it can be used for other future purposes as well.